src/uikit.cs

```diff
@@ -18848,13 +19595,1736 @@ interface UIPencilInteractionDelegate {
+	[DisableDefaultCtor]
+	interface UIWindowScene {
+
+		[Export ("initWithSession:connectionOptions:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UISceneSession session, UISceneConnectionOptions connectionOptions);
+
+		[Export ("screen")]
+		UIScreen Screen { get; }
+
+		[NoTV]
+		[Export ("interfaceOrientation")]
+		UIInterfaceOrientation InterfaceOrientation { get; }
+
+		[Export ("coordinateSpace")]
+		IUICoordinateSpace CoordinateSpace { get; }
+
+		[Export ("traitCollection")]
+		UITraitCollection TraitCollection { get; }
+
+		[Export ("windows")]
+		UIWindow [] Windows { get; }
+
+		[NoTV]
+		[NullAllowed, Export ("statusBarManager")]
+		UIStatusBarManager StatusBarManager { get; }
+
+		// From UIWindowScene (UIScreenshotService)
+
+		[NullAllowed, Export ("screenshotService")]
+		UIScreenshotService ScreenshotService { get; }
+	}
+
+	interface IUIWindowSceneDelegate { }
+
+	[iOS (13,0), TV (13,0), NoWatch]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface UIWindowSceneDelegate : UISceneDelegate {
+
+		[NullAllowed, Export ("window", ArgumentSemantic.Strong)]
+		UIWindow Window { get; set; }
+
+		[NoTV]
+		[Export ("windowScene:didUpdateCoordinateSpace:interfaceOrientation:traitCollection:")]
+		void DidUpdateCoordinateSpace (UIWindowScene windowScene, IUICoordinateSpace previousCoordinateSpace, UIInterfaceOrientation previousInterfaceOrientation, UITraitCollection previousTraitCollection);
+
+		[NoTV]
+		[Export ("windowScene:performActionForShortcutItem:completionHandler:")]
+		void PerformAction (UIWindowScene windowScene, UIApplicationShortcutItem shortcutItem, Action<bool> completionHandler);
+
+		[Export ("windowScene:userDidAcceptCloudKitShareWithMetadata:")]
+		void UserDidAcceptCloudKitShare (UIWindowScene windowScene, CKShareMetadata cloudKitShareMetadata);
+	}
+
+	[iOS (13,0), TV (13,0), NoWatch]
+	[BaseType (typeof (UISceneDestructionRequestOptions))]
+	interface UIWindowSceneDestructionRequestOptions {
+
+		[Export ("windowDismissalAnimation", ArgumentSemantic.Assign)]
+		UIWindowSceneDismissalAnimation WindowDismissalAnimation { get; set; }
+	}
+
+	[TV (13,0), iOS (13,0), NoWatch]
+	[BaseType (typeof (NSObject))]
+	interface UITabBarItemAppearance : NSCopying, NSSecureCoding {
+
+		[Export ("initWithStyle:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UITabBarItemAppearanceStyle style);
+
+		[Export ("configureWithDefaultForStyle:")]
+		void ConfigureWithDefault (UITabBarItemAppearanceStyle style);
+
+		[Export ("normal", ArgumentSemantic.Strong)]
+		UITabBarItemStateAppearance Normal { get; }
+
+		[Export ("selected", ArgumentSemantic.Strong)]
+		UITabBarItemStateAppearance Selected { get; }
+
+		[Export ("disabled", ArgumentSemantic.Strong)]
+		UITabBarItemStateAppearance Disabled { get; }
+
+		[Export ("focused", ArgumentSemantic.Strong)]
+		UITabBarItemStateAppearance Focused { get; }
+	}
+
+	// TODO: The following break the registrar with:
```

Comment:
file (or link to) an issue - and include the stack trace there (not here)

Reply:
Linked: https://github.com/xamarin/xamarin-macios/issues/6567